ArtOfIllusion/src/artofillusion/ArtOfIllusion.java

```diff
@@ -272,7 +273,7 @@ public static void newWindow()
   }
 
   /** Create a new window for editing the specified scene. */
-
+  @SuppressWarnings("CallToPrintStackTrace")
```

Comment:
If you're really certain the warning is inappropriate, it's probably a reasonable thing to do.